components/zoom_admin/zoom_admin.app.mjs

```diff
@@ -0,0 +1,333 @@
+      optional: true,
+    },
+    country: {
+      type: "string",
+      label: "Country",
+      description: "Registrant’s country. The value of this field must be in two-letter abbreviated form and must match the ID field provided in the [Countries](https://marketplace.zoom.us/docs/api-reference/other-references/abbreviation-lists#countries) table.",
+      optional: true,
+      options: zoomCountries,
+    },
+    pageSize: {
+      type: "integer",
+      label: "Page Size",
+      description: "The number of records returned within a single API call. Defaults to `30`",
+      optional: true,
+      min: 1,
+      max: 300,
+    },
+    pageNumber: {
+      type: "integer",
+      label: "Page Number",
+      description: "The page number of the current page in the returned records.",
+      optional: true,
+      min: 1,
+    },
+    nextPageToken: {
+      type: "string",
+      label: "Next Page Token",
+      description: "The next page token is used to paginate through large result sets. A next page token will be returned whenever the set of available results exceeds the current page size. The expiration period for this token is 15 minutes.",
+      optional: true,
+    },
+    webinar: {
+      type: "string",
+      label: "Webinar",
+      optional: true,
+      description: "The Webinar ID",
+      async options({
+        prevContext,
+        page,
+      }) {
+        if (!prevContext.nextPageToken && page > 0) {
+          return [];
+        }
+
+        const {
+          webinars,
+          next_page_token: nextPageToken,
+        } = await this.listWebinars({
+          nextPageToken: prevContext.nextPageToken,
+          pageSize: 2,
+        });
+        if (!webinars.length) {
+          return [];
+        }
+        const rawOptions = webinars.map((w) => ({
+          label: w.topic,
+          value: {
+            label: w.topic,
+            value: w.id,
+          },
+        }));
+        const options = sortBy(rawOptions, [
+          "label",
+        ]);
+
+        return {
+          options,
+          context: {
+            nextPageToken,
+          },
+        };
+      },
+    },
+  },
+  methods: {
+    _apiUrl() {
+      return "https://api.zoom.us/v2";
+    },
+    _getHeaders() {
+      return {
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+        "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
+        "user-agent": "@PipedreamHQ/pipedream v0.1",
+      };
+    },
+    _getAxiosParams(opts = {}) {
+      return {
+        ...opts,
+        url: this._apiUrl() + opts.path,
+        headers: this._getHeaders(),
+      };
+    },
```

Comment:
as a suggestion you can also omit path variable from axios config properties like this
```javascript
    _getAxiosParams(opts = {}) {
      const {
        path,
        ...otherOpts,
      } = opts;
      return {
        ...otherOpts,
        url: this._apiUrl() + path,
        headers: this._getHeaders(),
      };
    },
```